biz-scenario work/1 table.sql

```diff
@@ -0,0 +1,53 @@
+use CostumeDB
+go
+
+drop table if EXISTS dbo.Costume 
+GO
+create table dbo.Costume(
+    CostumeId int not null IDENTITY PRIMARY KEY,
+    BuyerName VARCHAR(50) not null, 
+    OwnerName VARCHAR(50) not null 
+        CONSTRAINT c_Owner_Name_cannot_be_blank CHECK(ownername <> ''),
+    ClientName VARCHAR(50) not null 
+        CONSTRAINT c_Client_Name_cannot_be_blank CHECK(ClientName <> ''),
+    RealtorName VARCHAR(50) not null 
+        CONSTRAINT c_Realtor_Name_cannot_be_blank CHECK(realtorname <> ''),
+    AddressNumber int not null
+        CONSTRAINT c_Address_Number_cannot_be_0 CHECK(addressnumber <> 0),
+    Street VARCHAR(30) not null 
+        CONSTRAINT c_Street_cannot_be_blank CHECK(street <> ''),
+    Town VARCHAR(30) not null 
+        CONSTRAINT c_Town_cannot_be_blank CHECK(town <> ''),
+    HouseType VARCHAR(15) not null 
+        CONSTRAINT c_House_Type_must_be_in_bilevel_colonial_ranch_splitlevel_duplex_townhouse_vacant_land_apartment 
+            CHECK(HouseType in('bi-level', 'colonial', 'ranch', 'split-level', 'duplex', 'townhouse', 'vacant land', 'apartment')),
+    HouseSquareFootage int not NULL 
+        CONSTRAINT c_House_Square_Footage_cannot_be_0 CHECK(HouseSquareFootage <> 0),
+    LotSquareFootage int not NULL 
+        CONSTRAINT c_Lot_Square_Footage_cannot_be_0 CHECK(LotSquareFootage <> 0),
+    NumberBathrooms int not NULL
+        CONSTRAINT c_Number_Bathrooms_cannot_be_0 CHECK(NumberBathrooms <> 0),
+    NumberBedrooms int not NULL
+        CONSTRAINT c_Number_Bedrooms_cannot_be_0 CHECK(numberbedrooms <> 0),
+    DateonMarket date not null
+        CONSTRAINT c_Date_on_Market_cannot_be_blank CHECK(DateonMarket <> ''),
+        CONSTRAINT c_Date_on_Market_must_be_before_current_date CHECK(DateonMarket < getdate()),
+    DateClosed DATE NULL DEFAULt ''
```

Comment:
BS: Constraint should also allow the closing date to be on the same day it was put on the market.